.travis.yml

```diff
@@ -4,6 +4,12 @@ language: go
 go:
   - 1.x
 
+services:
+  - postgresql
```

Comment:
You have docker compose, just user `docker` here!
And on the before_script run `docker-compose up`